cogs/wordfilter/testWordFilter.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from .wordfilter import (
+    _filterWord,
+    _isAllFiltered,
+    _isOneWord,
+)
+import re
+
+
+class TestHelperFunctions:
+    @pytest.mark.parametrize(
+        ["inputPhrase", "toFilter", "result"],
+        [
+            ("I am cool", "am", "I `**` cool"),
+            (
+                "https://discord.gg/testing123",
+                r"(discord.gg\/)(?!YcW3AtX)(?!MmpqtqD)[a-z,A-Z,0-9]*",
```

Comment:
I don't think we are catching commas?
And not taking in the context behind this RegEx into account, we can drop the first capture group also, since there's no logic that deals with specific captures.
```py
r"(?:discord.gg\/)(?!YcW3AtX)(?!MmpqtqD)[A-Za-z0-9]*"
```
Of course, since it is a test case so as far as that is concerned, this RegEx pattern has no problem.

Reply:
This was pulled straight from our regex list. I'll amend it in both places in a bit.